### ISSUE TITLE
Repeated Background Music Event restarts Music

### ISSUE THREAD

**Issue by CakeVR** [dialogic-godot/dialogic#2001]
**Is your feature request related to a problem? Please describe.**
If you start playing music A and reach another event that starts playing music A as well, music A restarts.

**Describe the solution you'd like**
Sometimes, when jumping or looping, the music playing is important instead of the music starting from anew.
I want the Music Event to have an option to prevent restarting the same audio resource from playing from its beginning.

Adding a property to the event "Restart if Resource plays already": `true` and `false` (default).

**Alternative Solution:**
I cannot think of a good reason when people may want to restart the audio to begin with, changing the behaviour may work as well.

**Comment by Jowan-Spooner** [comment]
Yeah I think for now I would just change the behaviour. I want to avoid any setting which I'm not sure people will actually use.

**Comment by CakeVR** [comment]
I will take a look a this then!
